recipes/dcase2023_task4_baseline/local/resample_folder.py

```diff
@@ -36,11 +37,24 @@ def resample(audio, orig_fs, target_fs):
     for c in range(audio.shape[0]):
         tmp = audio[c].detach().cpu().numpy()
         if target_fs != orig_fs:
-            tmp = librosa.resample(tmp, orig_fs, target_fs)
+            tmp = librosa.resample(tmp, orig_sr=orig_fs, target_sr=target_fs)
         out.append(torch.from_numpy(tmp))
     out = torch.stack(out)
     return out
 
+def resample_file(tuple_args, target_fs=16000):
```

Comment:
Tell me if you prefer the last version I submitted. Because it's not obvious to pass multiple arguments to a worker in a multiprocessing Pool, I had to use this workaround.